docs/Configuring-Orion/Using-PostgreSQL.md

```diff
@@ -0,0 +1,57 @@
+description: Storing Information with PostgreSQL 
+<!--- END of page meta data -->
+
+# Store Information with PostgreSQL
+
+PostgreSQL is an object-relational database management system you can use to store key values for Orion.
+
+## Prerequisites
+
+* [PostgreSQL](https://www.postgresql.org/)
+
+## Set up your PostgreSQL database
+
+1. In your database, create the `store` table by using the provided DDL script, as shown in the following example: 
+
+    `psql -h HOST-NAME -U USER-NAME -d DATABASE-NAME -f ./docs/Configuring-Orion/database/postgres_storage.sql` 
+    
+2. In your database, specify the users for the `store` table and assign permissions to them.
+
+## Configure Orion to use PostgreSQL
+
+In your [Orion configuration file](Configuration-File.md), specify `postgresql` for the `storage` property, as shown in the following example. Substitute your own values for `DATABASE-NAME`, `USER-NAME`, and `PASSWORD`. 
+
+  ```
+  storage="sql:jdbc:postgresql://localhost:5432/<DATABASE-NAME>?user=<USER-NAME>&password=<PASSWORD>"
```

Comment:
Does localhost:5432 also need replacing here?  Is that value specific to your setup?

Reply:
Changed these to variables